docker/README.md

```diff
@@ -23,6 +23,18 @@ $ docker run --name webapp -d \
     ghcr.io/thepalaceproject/circ-webapp:main
 ```
 
+If the database and ElasticSearch(ES) are running in containers, use the --link option to let the webapp docker container
```

Comment:
Need to update the build commands in the README file to reflect the new Dockerfile. They are referencing Dockerfile.webapp etc.

Reply:
DOEN